### PR TITLE
Fix some issues

### LINKED ISSUE #24: Add all softwares listed under AnythingMC
- Look at all the softwares listed under [AnythingsMC's software list](https://anythingmc.xyz/server/info/lists/server_software/) and the non-premium ones

### LINKED ISSUE #92: "Lost" category
## Idea
A lost category would contain all the softwares that don't have a repository or download link available.

## Why?
Currently lost softawres are included in the other files which makes it hard to differentiate between abandoned and lost softwares.
If you think you found a software fitting your needs, you might find out it's lost to time.

## Problems/Discussions
We'd have to decide if softwares like Cauldron, where there is no official repo anymore, should count as lost.

### LINKED ISSUE #94: The plural of "software" is "software"
The plural of "software" is "software", not "softwares".
As this hasn't become obvious to anyone yet, it seems to be a minor issue, but it might be worth changing the repo and names.

### LINKED ISSUE #98: [SOFTWARE] Thunderbolt
**What is the name of the software?**
[Thunderbolt](https://github.com/PlazmaMC/Thunderbolt)

**Describe what the server is in 1-2 sentences.**
Fork of PlazmaBukkit with experimental changes.

**What type of mods/plugins does the software support?**
Bukkit/Spigot/Paper/Purpur

**What version does the software support?**
1.20.6

**Additional information**
Experimental fork, testing ground for PlazmaBukkit

### PR DESCRIPTION
**Pull request by TonimatasDEV**
Add all software of AnythingsMC, create the LOST category, change "softwares" to "software" and many fixes.

Closes #94 
Closes #92 
Closes #24 
Close #98 

**Add (25):**
- Bravo
- CraftBukkit++
- Creativity
- Crocus
- DytanicSpigot
- FeatherSpigot
- Fish
- hCraft 2
- Myne2
- NessusMS
- Nostalgia
- Quartz
- QuartzPowered
- Quesadilla
- RainForest
- RubberBukkit
- TridentSDK
- Rubbercord
- TableCloth
- Rainbow
- Wolf In a Bukkit
- Torch
- Bamboo
- Sink-MC
- JTE-Freebuild

### CODE REVIEW

**Comment by TonimatasDEV** [comment]
@LeStegii Done. When you have time, check it.

File: OTHERS.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Others
-This list contains softwares that are related to minecraft servers but are no servers or proxies.
+This list contains software that are related to minecraft servers but are no servers or proxies.
```
Comment (LeStegii) [inline OTHERS.md:2]: Shouldn't this be "[...] software which is related to [...] but isn't server or proxy related." or something like this?

File: RECOMMEND.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
 # Recommended
 Find below our recommended software solutions tailored to your needs. Please note that while we strive to offer the best suggestions, individual preferences may vary.
 
-**Note:** Some softwares are outdated and are no longer being actively developed. They may still be useful, but they aren't supported and most likely contain security vulnerabilities, like Log4Shell, besides worse performance and compatibility. It's advisable to use newer, more efficient versions.
+**Note:** Some software are outdated and are no longer being actively developed. They may still be useful, but they aren't supported and most likely contain security vulnerabilities, like Log4Shell, besides worse performance and compatibility. It's advisable to use newer, more efficient versions.
```
Comment (LeStegii) [inline RECOMMEND.md:4]: "are" would have to be changed to "is"

File: RECOMMEND.md [hunk 3]
```diff
@@ -23,11 +23,12 @@ Find below our recommended software solutions tailored to your needs. Please not
 - Servers running **1.16.5** can use either [Mohist](java/MODS+PLUGINS.md#-mohist) or [Arclight](java/MODS+PLUGINS.md#-arclight).
 - Utilize [CatServer](java/MODS+PLUGINS.md#-catserver) for **1.12.2** servers.
 - For **1.7.10**, consider [Crucible](java/MODS+PLUGINS.md#-crucible).
-- Fabric mod and plugin integration can be achieved with [Mohist Banner](java/MODS+PLUGINS.md#-banner) or [Cardboard](java/MODS+PLUGINS.md#-cardboard-bukkit-for-fabric).
+- Fabric mod and plugin integration can be achieved with [Banner](java/MODS+PLUGINS.md#-banner) or [Cardboard](java/MODS+PLUGINS.md#-cardboard).
 
 ### 🪨 Minecraft Bedrock
+- Create a pure Minecraft experience without modifications, opt for the [Vanilla](bedrock/VANILLA.md#-vanilla) server.
```
Comment (LeStegii) [inline RECOMMEND.md:29]: The grammar seems to be wrong here

File: RECOMMEND.md [hunk 4]
```diff
@@ -23,11 +23,12 @@ Find below our recommended software solutions tailored to your needs. Please not
...
 - Create a Minecraft Bedrock Server with plugins using Spigot or Paper along with [Geyser](OTHERS.md#-geysermc).
 
 ### 🌐 Minecraft Networks
-- Connect multiple Java Edition servers using [Velocity](java/PROXIES.md#-velocity).
+- Connect multiple Java Edition servers using [Velocity](java/PROXIES.md#-velocity) or [Bungeecord](java/PROXIES.md#-bungeecord).
```
Comment (LeStegii) [inline RECOMMEND.md:33]: I don't know if we should recommend bungeecord.

File: bedrock/PLUGINS.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Minecraft Bedrock Plugins Software.
-This list contains Minecraft Bedrock plugins server softwares.
+This list contains Minecraft Bedrock plugins server software.
 
-Since most of the softwares do not support all vanilla features, it's recommended to use [GeyserMC](https://github.com/UeberallGebannt/server-softwares/blob/master/OTHERS.md#-geysermc) (Bedrock to Java Bridge) instead if you want to play vanilla.
+Since most of the software do not support all vanilla features, it's recommended to use [GeyserMC](/OTHERS.md#-geysermc) (Bedrock to Java Bridge) instead if you want to play vanilla.
```
Comment (LeStegii) [inline bedrock/PLUGINS.md:4]: "do" would have to be changed to "does"

File: bedrock/VANILLA.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
 # Minecraft Bedrock Vanilla Software
-This list contains Minecraft Bedrock vanilla server softwares.
+This list contains Minecraft Bedrock vanilla server software.
 
 # ✔️ Active Development
-### [🌍 Vanilla Bedrock (Bedrock Dedicated Server)](https://www.minecraft.net/en-us/download/server/bedrock)
+### [🌍 Vanilla](https://www.minecraft.net/en-us/download/server/bedrock)
```
Comment (LeStegii) [inline bedrock/VANILLA.md:5]: I think "Bedrock Dedicated Server" should stay

File: java/LOST.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+### ⬛ TridentSDK
+- **Version:** 1.8
+- **Author:** ?
+- **Fork:** -
+- **Description:** ?
+
+### ⬛ Rubbercord
+- **Version:** ?
+- **Author:** ?
```
Comment (LeStegii) [inline java/LOST.md:114]: I think this has been made by md_5

File: java/MODS+PLUGINS.md [hunk 8]
```diff
@@ -53,14 +53,14 @@ This list contains Minecraft Java mods/plugins server softwares.
...
 ### [🚩 Banner](https://github.com/MohistMC/Banner)
-- **Version:** 1.19.4, 1.20.1, 1.20.2 (Experimental), 1.20.4-1.20.6 (Experimental)
+- **Version:** 1.19.4, 1.20.1, 1.20.2 (Experimental), 1.20.4-1.21 (Experimental)
 - **Author:** MohistMC
 - **Fork:** Fabric/CraftBukkit/Spigot/Paper/Mohist/Arclight --> Banner
 - **Description:** Fabric Mod that allows the use of Bukkit/Spigot/Paper Plugins.
 - **Plugins:** Bukkit, Spigot, Paper
 - **Mods:** Fabric
 
-### [🧻 Cardboard (Bukkit for Fabric)](https://www.curseforge.com/minecraft/mc-mods/cardboard)
```
Comment (LeStegii) [inline java/MODS+PLUGINS.md:63]: The official curseforge name still includes "Spigot for Fabric" so we could include this as well

File: java/PLUGINS.md [hunk 9]
```diff
@@ -172,6 +172,13 @@ This list contains Minecraft Java plugins server softwares.
 - **Plugins:** Bukkit
 
 # ❌ Inactive Development
+### [⬛ Rainbow](https://ci.codecrafter47.de/job/Rainbow/)
+- **Version:** 1.13.2
+- **Author:** codecrafter47
+- **Fork:** -
+- **Description:** Rainbow was an attempt at overthrowing Bukkit with a new platform.
+- **Plugins:** -
+
```
Comment (LeStegii) [inline java/PLUGINS.md:181]: As this doesn't supprt plugins, why is it in the plugins category?

**Review by LeStegii — CHANGES_REQUESTED** [review]

File: java/PLUGINS.md (hunk 9 above)
Comment (LeStegii) [reply java/PLUGINS.md:181]: Also, from what I can see, the ci links to a github repository which we should link instead

File: java/MODS+PLUGINS.md (hunk 8 above)
Comment (TonimatasDEV) [reply java/MODS+PLUGINS.md:63]: In GitHub, the logo and the slug are only Cardboard

File: java/PLUGINS.md (hunk 9 above)
Comment (TonimatasDEV) [reply java/PLUGINS.md:181]: > As this doesn't supprt plugins, why is it in the plugins category?

Yes it supports.
https://ci.codecrafter47.de/job/Rainbow/org.projectrainbow$PluginExample/

**Comment by TonimatasDEV** [comment]
Done!

File: java/LOST.md
Comment (lunofe) [inline java/LOST.md:1]: AnchorMC might just have moved, looks like the original author: https://github.com/PassTheMayo/AnchorMC

**Review by lunofe — CHANGES_REQUESTED** [review]
AnchorMC might just have moved, looks like the original author: https://github.com/PassTheMayo/AnchorMC

**Comment by LeStegii** [comment]
> AnchorMC might just have moved, looks like the original author: https://github.com/PassTheMayo/AnchorMC

Good catch!

**Comment by lunofe** [comment]
One more thing; should the _lost_ category actually be a Java exclusive, instead of a general one, just like _others_?

https://github.com/LeStegii/server-softwares/blob/949aabc3e4f53f4669d96aac8b29c017968ed962/java/LOST.md?plain=1#L1-L2

**Comment by LeStegii** [comment]
> One more thing; should the _lost_ category actually be a Java exclusive, instead of a general one, just like _others_?
> 
> https://github.com/LeStegii/server-softwares/blob/949aabc3e4f53f4669d96aac8b29c017968ed962/java/LOST.md?plain=1#L1-L2

It depends. It would make sense to split java and bedrock lost softwares.

**Comment by lunofe** [comment]
But is it? I feel like if it's lost, you don't really need a separation because you're most likely just checking the category to learn some trivia,

**Comment by LeStegii** [comment]
> But is it? I feel like if it's lost, you don't really need a separation because you're most likely just checking the category to learn some trivia,

Maybe we could add something like `- Type: Bedrock/Java/Other` than

**Comment by TonimatasDEV** [comment]
Done! Moved LOST to general and update AnchorMC

**Comment by TonimatasDEV** [comment]
I added Thunderbolt now.

**Comment by LeStegii** [comment]
@lunofe Could you hit me up when the gh pages are ready so that I can merge this PR?

**Comment by lunofe** [comment]
@LeStegii can be merged 👍🏻

**Review by LeStegii — APPROVED** [review]

**Pull request merged**